platform/rct/hal/imu_stm32/imu_device_stm32.c

```diff
@@ -39,7 +39,7 @@
 #define IMU_TASK_PRIORITY	(tskIDLE_PRIORITY + 2)
 #define IS_9150_ADDR            0x68
 
-static struct is9150_all_sensor_data sensor_data[2];
+static struct is9150_all_sensor_data sensor_data[2] = {0};
```

Comment:
So, research tells me that ld does not have the ability to distribute objects between multiple bss sections. Apparently some commercial linkers can do this, but not GNU's ld. 

The links below are my research - first 3 links have the strongest discussions around this; bottom links to documentation does not show any information about combining multiple regions. 

At the moment, I think that the {0} initializer hint is the most portable solution to achieving the goals we want.  When we pull the predictive timer out of RaceCapture/Track's firmware and move it into the app, we'll recover a big chunk of RAM and might be able to unwind most if not all of this. 
-----------------------------------------------
https://community.arm.com/iot/embedded/f/discussions/6266/linker-instruction-to-split-bss-section-into-two-different-ram-banks

https://mcuoneclipse.com/2013/04/14/text-data-and-bss-code-and-data-size-explained/#comment-72754

https://e2e.ti.com/support/development_tools/compiler/f/343/t/104507
https://community.nxp.com/thread/426299


https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/4/html/Using_ld_the_GNU_Linker/sections.html#OUTPUT-SECTION-REGION

ftp://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html#SEC19